lib/add-tests.coffee

```diff
@@ -220,14 +220,19 @@ addTests = (raml, tests, hooks, parent, callback, testFactory, apiBaseUri) ->
             headers[header.name()] = header.example().value()
           else if header.examples().length > 0
             headers[header.name()] = header.examples()[_.random(0, header.examples().length-1)].value()
-          else if header.type()[0] of types && (types[header.type()[0]].example()? || types[header.type()[0]].examples().length > 0)
-            type = types[header.type()[0]] 
+          else if header.type()[0] of types && \
```

Comment:
I've never used this style of new line in coffeescript before, does it definitely compile properly?